lib/queryBuilder/QueryBuilder.js

```diff
@@ -1096,9 +1078,13 @@ Object.defineProperties(QueryBuilder.prototype, {
   }
 });
 
-function parseRelationExpression(modelClass, expr) {
+function parseRelationExpression(modelClass, exp, filters) {
   try {
-    return RelationExpression.parse(expr);
+    const expr = RelationExpression.parse(exp);
+    if (filters) {
+      expr.filters = filters;
+    }
```

Comment:
@koskimas we could consider moving this part to `RelationExpression.parse()` with a second filters argument. Maybe it's better to do the setting of `filters` there than here, since it's a property of `RelationExpression`

Reply:
@koskimas oops, this came after the merge... But I think it's worth considering.